src/app/helpers/lazyImageLoading.js

```diff
@@ -0,0 +1,33 @@
+const createLazyListener = (selector, dataset) => () => {
+  let images = document.querySelectorAll(selector);
+  if (images.length) {
+    images = [].slice.call(images);
```

Comment:
if you want to clone an array, that construction should be fine [... images]
Even better to use Array.from(images)

Reply:
Arra.from is new, that's why I forget about it, thx!
I have copied this func, then changed

Reply:
> if you want to clone an array, that construction should be fine [... images]
> Even better to use Array.from(images)

Arra.from is new, that's why I forget about it, thx!
I have copied this func, then changed